reference/compiling_for_android.rst

```diff
@@ -16,93 +16,81 @@ Also, you still need to do all the steps mentioned in the
 :ref:`doc_exporting_for_android` tutorial before attempting your custom
 export template.
 
-Requirements
-------------
+A. Requirements
+---------------
 
 For compiling under Windows, Linux or OSX, the following is required:
 
--  Python 2.7+ (3.0 is untested as of now).
--  SCons build system.
+-  Python 2.7+ [3.0 is untested as of now]
+-  pywin32 [Optional, Windows only. Applied when -j(number of cores) specified]
+-  SCons build system
 -  Android SDK version 19 [Note: Please install all Tools and Extras of sdk manager]
 -  Android build tools version 19.1
--  Android NDK
--  Gradle
--  OpenJDK 6 or later (or Oracle JDK 6 or later)
+-  Android NDK r10e or later
+-  Gradle [will be downloaded and installed when compiling for android if it doesn't exist]
+-  JDK 6+ (Oracle's releases) or OpenJDK 6+
```

Comment:
Why? I think it better to put forward the open source implementation. This change is gratuitous.